pubspec.yaml

```diff
@@ -5,6 +5,7 @@ environment:
 workspace:
   - packages/artifact_proxy
   - packages/discord_gcp_alerts
+  - packages/flutter_version_resolver
```

Comment:
might be nice to split this into `flutter_version_parser` and `flutter_version_solver` 🤷 
we can always do that later but I think there are conceptually two independent pieces here at least the way I was approaching this problem.

Reply:
Yeah, we can split this out in the future when we actually add any sort of resolution logic, rather than just rummaging through various files to find an exact version number.